devicon.json

```
@@ -2354,20 +2354,26 @@
     {
         "name": "ember",
         "altnames": [
-            "emberjs"
+            "ember.js"
```

Emberjs is also used a lot. For example on twitter. A few third party learning platforms also call it emberjs
```suggestion
            "ember.js",
            "emberjs"
```

---

I'm ok with it! 🙂